index.js

```diff
@@ -4,6 +4,8 @@ export const separator = Symbol('separator');
 
 export const isDarkMode = process.env.XBARDarkMode === '1';
 
+export const isxbar = process.env.__CFBundleIdentifier === 'com.xbarapp.app';
```

Comment:
Is `__CFBundleIdentifier` documented anywhere? Can be rely on it not disappearing in the future?

Reply:
I had no idea that was a thing. Thanks for elaborating.